spec/resources/workbooks_spec.rb

```diff
@@ -184,6 +161,74 @@ def find_or_publish_workbook(name)
     end
   end
 
+  describe '#permissions' do
+    it 'can retrieve multiple permissions for a workbook' do
+      workbook = find_or_publish_workbook('testpublish')
+      expected = [{
+        grantee_type: 'group',
+        grantee_id: all_users_group['id'],
+        capabilities: {
+          Read: true,
+          ExportData: true,
+          ViewComments: true,
+          AddComment: true
+        }
+      }, {
+        grantee_type: 'group',
+        grantee_id: test_group['id'],
+        capabilities: {
+          ChangePermissions: false
+        }
+      }, {
+        grantee_type: 'user',
+        grantee_id: admin_user['id'],
+        capabilities: {
+          ChangePermissions: false
+        }
+      }]
+      actual = client.workbooks.permissions(workbook_id: workbook['id'])
+      expect(actual).to eq expected
+    end
+
+    it 'returns an array for even a single permission' do
+      workbook = find_or_publish_workbook('testpublish')
+
+      # use up the VCR recording from the previous spec
```

Comment:
Unfortunately, I don't think there's a good way to address this the way the tests are currently written. It may be worth trying to make recording new episodes simpler, but as a separate PR.